datadog_operator/assets/dashboards/operator_overview.json

```diff
@@ -0,0 +1,1987 @@
+                                            "aggregator": "last"
+                                        }
+                                    ]
+                                }
+                            ],
+                            "custom_links": [],
+                            "precision": 0
+                        },
+                        "layout": {
+                            "x": 8,
+                            "y": 2,
+                            "width": 2,
+                            "height": 2
+                        }
+                    },
+                    {
+                        "id": 1642888553658669,
+                        "definition": {
+                            "title": "CLC Worker Pods",
+                            "title_size": "16",
+                            "title_align": "left",
+                            "type": "query_value",
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "response_format": "scalar",
+                                    "queries": [
+                                        {
+                                            "query": "sum:kubernetes.pods.running{$scope,$cluster,kube_app_managed_by:datadog-operator,kube_deployment:*,kube_app_instance:datadog-agent-cluster-checks-runner}",
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "aggregator": "last"
+                                        }
+                                    ]
+                                }
+                            ],
+                            "custom_links": [],
+                            "precision": 0
+                        },
+                        "layout": {
+                            "x": 10,
+                            "y": 2,
+                            "width": 2,
+                            "height": 2
+                        }
+                    }
+                ]
+            },
+            "layout": {
+                "x": 0,
+                "y": 5,
+                "width": 12,
+                "height": 5
+            }
+        },
+        {
+            "id": 6439676071842866,
+            "definition": {
+                "title": "Operator Errors",
+                "background_color": "green",
+                "show_title": true,
+                "type": "group",
+                "layout_type": "ordered",
+                "widgets": [
+                    {
+                        "id": 6952001278071111,
+                        "definition": {
+                            "title": "Operator CLB by Cluster",
```

Comment:
Non blocking nit. CLB I'm guessing is CrashLoopBackOff? For clarity it would be better to use the term unless CLB is a widely known term.

Reply:
Thanks for pointing this out. Indeed, `clb kubernetes` gives hits on load balancers, no crashloops.